src/current/cockroachcloud/metrics.md

```diff
@@ -28,8 +28,8 @@ Depending on your CockroachDB {{ site.data.products.cloud }} deployment, the **M
 | [SQL]({% link cockroachcloud/metrics-sql.md %}) | ✔ |  |  |
 | [Changefeeds]({% link cockroachcloud/metrics-changefeeds.md %}) | ✔ |  |  |
 | [Row-Level TTL]({% link cockroachcloud/metrics-row-level-ttl.md %}) | ✔ |  |  |
-| [Monitor SQL Activity]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) |  | ✔ | ✔ |
-| [Identify SQL Problems]({% link cockroachcloud/metrics-identify-sql-problems.md %}) |  | ✔ | ✔ |
+| [Monitor SQL Activity]({% link cockroachcloud/metrics-monitor-sql-activity.md %}) | available on [SQL]({% link cockroachcloud/metrics-sql.md %}) tab | ✔ | ✔ |
```

Comment:
"Available" should be capitalized.

But also, I'm confused what this means. Is this still accessible through the Metrics page? If so, I think a checkmark might still be clearest.

Reply:
updated with "Same graphs are available on Metrics page SQL tab"